commodore/cli.py

```diff
@@ -417,6 +419,46 @@ def component_versions(
         click.echo(yaml.safe_dump(components))
 
 
+@inventory.group(
+    name="lint", short_help="Commands which lint the Commodore inventory structure"
+)
+@verbosity
+@pass_config
+def inventory_lint(config: Config, verbose):
+    config.update_verbosity(verbose)
```

Comment:
Could we make this command lint everything (i.e. execute all it's sub commands)?

I pretty much never want to just check for one type of issues. I would want commodore to just lint my inventory. I don't want to run a command to check if there are components without versions, then another one check for a different issue, etc.

Reply:
Not sure if that's possible with the current structure, but we can certainly refactor the CLI to just provide `inventory lint` and then options to enable or disable individual linters. Thinking about it again, that probably makes more sense than my current approach.

Reply:
Where would that linting be executed?

Reply:
Changed the CLI to just provide `inventory lint`. We can always add flags to enable or disable individual linters once we have more than one.